java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/core/JniDbzSourceHandler.java

```diff
@@ -58,7 +58,7 @@ public static void runJniDbzSourceThread(byte[] getEventStreamRequestBytes, long
     }
 
     public void start(long channelPtr) {
-        var runner = DbzCdcEngineRunner.newCdcEngineRunner(config);
```

Comment:
Prior to this PR, how can this pass compilation?

Reply:
Java can have same function name with different parameters.